dysql/pydantic_mappers.py

```diff
@@ -90,6 +90,13 @@ def _map_dict(self, current_dict: dict, record: sqlalchemy.engine.Row, field: st
 
     def _map_list_from_string(self, current_dict: dict, record: sqlalchemy.engine.Row, field: str):
         list_string = record[field]
+        if list_string is None:
+            # See note above for lists
+            return
+
+        if list_string:  # force it to be a string
+            list_string = str(list_string)
+
         if list_string and isinstance(list_string, str):
```

Comment:
won't this always be true now if the one above passes?

Reply:
yeah.